### DISCUSSION TITLE
instant reject

### DISCUSSION THREAD

**Discussion by DuperKnight** [community/community#57109] [GitHub Education]
### Hi everyone,

I’m having trouble with the GitHub Student Pack, I’m being instantly rejected even though I have a valid Student ID, I receive a message (attached image), I can’t contact support and I don’t know what to do.
![Screenshot 2023-06-03 220834](https://github.com/community/community/assets/64538870/aa711fc3-f84a-4442-8027-d85a06599b5a)

**Comment by karimdevelops** [comment]


**Reply by DuperKnight** [reply]
Thanks for the help, I think I won't be able to use my school ID, it's a bit blurry because I dropped it on water and the material is bad 😅

**Comment by samus-aran** [comment]
Hi @Dupernite 👋 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

**Reply by acastkat** [reply]
Hi, I'm having the same issue. I tried to apply again but GitHub doesn't let me, how can I upload additional evidence or apply again?

**Reply by samus-aran** [reply]
Hi @andreacastkatsini  👋

We’re happy you’re here but let’s not divert this discussion as this conversation is off-topic. Please open up a separate discussion so that we can look at your query there. 👍
